istio/README.md

```diff
@@ -24,17 +24,26 @@ Edit the `istio.d/conf.yaml` file (in the `conf.d/` folder at the root of your [
 
 Add this configuration block to your `istio.d/conf.yaml` file to start gathering your Istio Metrics:
```

Comment:
```suggestion
```
Should this be removed too?

Reply:
I can reword it to introduce the two options?

Reply:
Sure!